company_linkage/sql/organizations.sql

```diff
@@ -1,18 +1,30 @@
 WITH sp_500 AS (
   SELECT DISTINCT
-    CSET_id
+    new_cset_id
   FROM
     parat_input.groups
   where
     name = "S&P 500"
 ),
 global_500 AS (
   SELECT DISTINCT
-    CSET_id
+    new_cset_id
   FROM
     parat_input.groups
   where
     name = "Global 500"
+),
+mapped_parents AS (
+  SELECT
+    parentage.new_cset_id,
+    COALESCE(legacy_cset_id, 4000+parent_id) AS parent_id,
```

Comment:
Okay, so just trying to understand what's happening here:

We're linking the parent id (which has been changed here already by annotators to the new CSET ids? Is that correct?) to the organizations table, to find the org info about the parents. Then, we're finding the legacy cset id of the parent, and setting the parent id to be either the legacy id or 4000 + the new id. Meanwhile, we're leaving the cset id itself as the new_cset_id of the org itself, for now.

Reply:
Yep - the parent id is the "new cset id" for the parent organization - we're changing it here to match the ids that will be used in `organizations`. We're keeping the new cset id around for joining purposes